apps/api/tests/test_urls.py

```diff
@@ -33,8 +33,8 @@ def test_responses(self):
         eq_(piston_response, drf_response)
 
     def test_responses_with_handler(self):
-        view = SwitchToDRF('User', with_handler=True)
-        request = self.factory.get(reverse('api.language', args=['2']))
+        view = SwitchToDRF('Addons', with_handler=True, with_viewset=True)
+        request = self.factory.get(reverse('api.addons'))
```

Comment:
It is related, in a tricky way: there was a hack (line 47 and 48) to make both APIs work at the same time in this test: one API needed an anonymous user, and the other one a logged in user. Instead of keeping this hack, and as the test is only making sure that the handlers are properly used, I chose to use another endpoint which was working with an anonymous user for both (it just returns an empty list of addons).